### DISCUSSION TITLE
Draw SVG to constrained Width/Height

### DISCUSSION THREAD

**Discussion by rlightner** [wieslawsoltes/Svg.Skia#248] [Q&A]
I need to draw the SVG to a canvas that has a specific width and height. Is there a way to have the SVG drawn centered to these constraints?

**Comment by wieslawsoltes** [comment]
Check how the Svg control does it.

**Reply by rlightner** [reply]
Are you referring to AvaloniaSvgSkiaSample? I'm not using xaml so I don't have access to the DrawingContext

**Reply by wieslawsoltes** [reply]
> Are you referring to AvaloniaSvgSkiaSample? I'm not using xaml so I don't have access to the DrawingContext

Just check how skpicture is rendering there
